recipe/meta.yaml

```diff
@@ -1,125 +1,85 @@
-{% set version = "2.12.3" %}
+{% set version = "2.13.1" %}
+{% set name = "pytensor" %}
 
 package:
-  # https://github.com/conda/conda-build/issues/3933
-  name: pytensor-suite
+  name: {{ name }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/p/pytensor/pytensor-{{ version }}.tar.gz
-  sha256: fa82d29ef0594e0a2751eaad40882dcd6af51333c28be0d79c1d0de9f3a81a86
-  patches:
+  git_url: https://github.com/pymc-devs/{{ name }}.git
+  git_rev: rel-{{ version }}
+  sha256: 2f704509abe811f9bca392b6848b1ed5b5011210fc19fd67d17e53f7f7362637
```

Comment:
any reason to download with git instead of using the github provided tarball?

https://github.com/pymc-devs/pytensor/archive/refs/tags/rel-2.13.1.tar.gz

Reply:
about tests missing in the pypi tarball, we probably want to report that upstream.

Reply:
the reason is that the tarball is missing a file called `unittest_tools.py` which is used in their own test framework. The file is not missing in the GitHub repo, hence the download from there. This way tests can successfully be run.
I will report to the maintainers.

Reply:
the file I linked to you contains exactly the same files as a git checkout, and prevents using git directly, which is usually better for packages.